client/plots/sampleView.js

```diff
@@ -567,8 +567,15 @@ class SampleView {
 			for (const sample of samples) {
 				const cellDiv = div.append('div').style('display', 'inline-block')
 				const wsiViewer = await import('./wsiviewer/plot.wsi.js')
-				wsiViewer.default(state.vocab.dslabel, cellDiv, this.app.opts.genome, sample.sampleName)
+				await wsiViewer.default(state.vocab.dslabel, cellDiv, this.app.opts.genome, sample.sampleName)
```

Comment:
I would throw the error and make the try catch here, if there is an error continue, nothing would be added to cellDiv